app/models/items/recurring_donation.rb

```diff
@@ -6,4 +6,34 @@ class RecurringDonation < Item
 
   def one_line_description ; end
   def description_for_audit_txn ; end
+  def monthly_amount
+    # Finds the first donation with the recurring_donation_id matching this instance's id
+    donation = Donation.find_by(recurring_donation_id: id)
+    donation ? donation.amount : 0  # Returns the donation amount if found, otherwise returns 0
```

Comment:
i feel like if donation is not found, monthly_amount should just throw an error or something rather than return 0, so that logical errors can be caught asap. its inconsequential for now, since monthly_amount is only used in total_to_date, but i imagine that in the future, when monthly_amount is being used in multiple places, its better to catch an explicit error than to let the code continue until completion with faulty logic. kind of like kicking the can down the road. what do you think about this?

Reply:
Agreed, what should error be?

Reply:
fixed